.gitignore

```diff
@@ -125,3 +125,16 @@ nCrunchTemp_*
 
 tests/FSharp.Compiler.Service.Tests/FSharp.CompilerService.SurfaceArea.netstandard.actual
 *.vsp
+/Z.fs
```

Comment:
Do we need to keep those added ignores?

Reply:
Some tests generate them, and I wasn't ready to go through the tests looking to find out where the files came from.